eth.gemspec

```diff
@@ -48,4 +48,7 @@ Gem::Specification.new do |spec|
 
   # scrypt for encrypted key derivation
   spec.add_dependency "scrypt", "~> 3.0"
+
+  # activesupport for smart contract support
+  spec.add_dependency "activesupport", "~> 7.0"
```

Comment:
where is this used?

Reply:
It was used in `present?` and `underscore` methods.
I will probably remove it later when it is no longer needed.

Reply:
I think those are both one-line methods that could be copied and pasted into `Eth::Util` or similar rather than pull in the entirety of activesupport

Reply:
Yes, that's what I was thinking. Keep dependencies as low as possible.

Reply:
Remove the dependency on activesupport and use the methods of Ruby itself.